### PR TITLE
fix(account): remove GetLogin

### PR DESCRIPTION
**Pull request by remyleone**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/scaleway/scaleway-cli/blob/master/CHANGELOG.md):

```release-note

```

### CODE REVIEW

File: internal/namespaces/init/init.go [hunk 1]
```diff
@@ -371,56 +371,11 @@ func promptCredentials(ctx context.Context) (string, error) {
 	}
```
Comment (kindermoumoute) [inline internal/namespaces/init/init.go:371]: Update prompted information "Enter a valid secret-key or an email" ==> "Enter a valid secret-key"

File: internal/namespaces/init/init.go [hunk 2]
```diff
@@ -371,56 +371,11 @@ func promptCredentials(ctx context.Context) (string, error) {
...
```
Comment (kindermoumoute) [inline internal/namespaces/init/init.go:372]: Can we ask for access key too?

**Review by kindermoumoute — CHANGES_REQUESTED** [review]

**Review by kindermoumoute — DISMISSED** [review]
LGTM

File: internal/namespaces/init/testdata/test-init-ssh-with-local-ed25519-key.cassette.yaml [hunk 3]
```diff
@@ -35,155 +43,115 @@ interactions:
...
       User-Agent:
-      - scaleway-sdk-go/v1.0.0-beta.6+dev (go1.14.4; darwin; amd64) cli-e2e-test
+      - scaleway-sdk-go/v1.0.0-beta.7+dev (go1.18.2; darwin; amd64) cli-e2e-test
     url: https://api.scaleway.com/account/v2alpha1/ssh-keys
     method: POST
   response:
-    body: '{"id":"255abc8e-4628-4d53-a497-e1bd23c06757","name":"","public_key":"ssh-ed25519
-      AAAAC3NzaC1lZDI1NTE5AAAAIJb42xh9l10D/u9imDFfLZ+U6KrZmr/F/qBClnmijCFF/qEehPJxK/3thmMiZg foobaz@foobaz","fingerprint":"256
-      MD5:ea:c1:a1:82:4b:41:2b:65:26:9c:18:27:da:0d:bd:91 foobar@foobar (ssh-ed25519)","created_at":"2020-06-10T13:03:44.732785Z","updated_at":"2020-06-23T14:33:35.687332Z","creation_info":{"address":"","user_agent":"","country_code":""},"organization_id":"951df375-e094-4d26-97c1-ba548eeb9c42"}'
+    body: '{"details":[{"argument_name":"public_key","help_message":"invalid key format:
```
Comment (remyleone) [inline internal/namespaces/init/testdata/test-init-ssh-with-local-ed25519-key.cassette.yaml:93]: @sweenu @kindermoumoute Do you know if additional checks are performed on your side now?

**Review by kindermoumoute — APPROVED** [review]

**Pull request merged**